src/components/mgt-person-card/mgt-person-card.ts

```diff
@@ -450,6 +488,39 @@ export class MgtPersonCard extends MgtTemplatedComponent {
    * @memberof MgtPersonCard
    */
   protected async loadState() {
+    const provider = Providers.globalProvider;
+    const graph = provider.graph.forComponent(this);
```

Comment:
This causes exceptions if `provider` is null. Move this line below the check for signed in user